test/libsolidity/ASTJSONTest.h

```diff
@@ -64,12 +85,9 @@ class ASTJSONTest: public TestCase
 		std::string const& _variation
 	) const;
 
+	std::vector<TestVariant> m_variations;
```

Comment:
Why not `m_variants`?
```suggestion
	std::vector<TestVariant> m_variants;
```